### ISSUE TITLE
TS73: JO Number Series is incorrect

### ISSUE THREAD

**Issue by HannahCastro** [vasavah/vsv-fli-grn#52]
**Describe the bug**
1. JO Series is incorrect. We endorsed the JO syntax on May 6, 2019.
2. JO Numbers also appear differently under PO Management and Job Order Modules. Under PO module, JO Number follows the "LOG" series. But under JO module, it shows the PO number instead.
3. There is no JO number when viewing the JO itself. It just shows the referenced PO.

**Expected behavior**
1. JO Number will be generated once canvasser submits JO.
2. JO Syntax is "LOG" + last 2 digits of year created + "-" + 6 digit series number (starting with 000001) ex. LOG19-000001.
3. In Job Order module, correct JO Number should appear. The PO Number referenced should be in an additional column (2nd column).
4. In viewing a specific JO, it should appear as "Viewing JO [JO Number] for [PO Number]."

**Screenshots**
![image](https://user-images.githubusercontent.com/51902789/64684189-3bddb500-d4b7-11e9-9924-e3520ba077d3.png)
![image](https://user-images.githubusercontent.com/51902789/64684441-b73f6680-d4b7-11e9-8d3d-547439430270.png)
![image](https://user-images.githubusercontent.com/51902789/64686254-f4592800-d4ba-11e9-8433-bae75b28ff42.png)

**Desktop (please complete the following information):**
 - OS: [Windows 7]
 - Browser [Google Chrome]
 - Version [75.0.3770.100]

**Comment by HannahCastro** [comment]
JO Number series is now correct. But in the JO list, the JO Number column is still the 2nd column.
![image](https://user-images.githubusercontent.com/51902789/69391955-56ca7200-0d0f-11ea-8d62-6c852990d4f9.png)
![image](https://user-images.githubusercontent.com/51902789/69391975-65b12480-0d0f-11ea-8247-387ded9459e0.png)

**Comment by HannahCastro** [comment]
JO Number series is correct and appears in the JO. Referenced PO is also attached in the JO created. We have decided to just retain the placement of the JO Number and PO Number columns in the Job Order listing.
![image](https://user-images.githubusercontent.com/51902789/69401285-a3be4080-0d2f-11ea-86f3-140cc1e9ade0.png)
![image](https://user-images.githubusercontent.com/51902789/69401306-afaa0280-0d2f-11ea-8992-0c32f1467878.png)

This issue is now resolved.
